### PR TITLE
Monday source documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for Monday source

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/event-streams/cloud-apps/monday.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  },
+}
+```
+
+RudderStack transforms the above payload into the following payload:
+
+```json
+{
+  "type": "track",
+  "event": "deletePulse",
```
Comment (yashasvibajpai) [inline docs/sources/event-streams/cloud-apps/monday.mdx:100]:
```suggestion
  "event": "Delete Pulse",
```
We had updated the event names to be like how they are in Rudderstack.

File: docs/sources/event-streams/cloud-apps/monday.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+| `type` | `properties.type` | 
+| `subscriptionId` | `properties.subscriptionId` |
+| `triggerUuid` | `properties.triggerUuid` |
+| `parentItemId` | `properties.parentItemId` |
+| `parentItemBoardId` | `properties.parentItemBoardId` |
+| `itemId` | `properties.itemId` |
+| `previousValue` | `properties.previousValue` |
+| `value` | `properties.value` |
+
+RudderStack supports ingesting the events related to the following Monday.com standard actions:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/cloud-apps/monday.mdx:55]: just checking if it needs to be "Monday.com" here?

File: docs/sources/event-streams/cloud-apps/monday.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+- A new update is posted.
```
Comment (ishachopra0401) [inline docs/sources/event-streams/cloud-apps/monday.mdx:57]:
```suggestion
- A new update is posted.
- A status is changed.
- A column is changed.
- A specific column is changed.
- An item is created.
- An item is deleted.
- An item name is changed.
- An item is archived.
- A subitem is created.
- A subitem is deleted.
- A new update is posted on a subitem.
- A subitem column is changed.
- A subitem name is changed.	
- A subitem is archived.
```

just rearranged to keep the similar ones together

**Review by ishachopra0401 — APPROVED** [review]

File: docs/sources/event-streams/cloud-apps/monday.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/sources/event-streams/cloud-apps/monday.mdx:55]: yep - the nav bar and the website seems to reference it as Monday.com everywhere

Screen Shot 2022-09-07 at 1 40 30 PM

File: docs/sources/event-streams/cloud-apps/monday.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply docs/sources/event-streams/cloud-apps/monday.mdx:57]: Perfect, thanks!

File: docs/sources/event-streams/cloud-apps/monday.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/sources/event-streams/cloud-apps/monday.mdx:55]: I'll make the name consistent throughout the doc.

File: docs/sources/event-streams/cloud-apps/monday.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+[monday.com](https://monday.com/) is a popular workflow management tool. It lets you create efficient cross-team workflows and automations for better collaboration and maximum productivity.
+
+This guide will help you set up monday.com as a source in RudderStack.
+
+## Getting started
+
+Follow these steps to set up your monday.com source in the RudderStack dashboard:
+
+1. Go to your [RudderStack dashboard](https://app.rudderstack.com/) and click **Add Source**. From the list of **Event Streams** sources, select **monday.com**.
```
Comment (ishachopra0401) [inline docs/sources/event-streams/cloud-apps/monday.mdx:15]:
```suggestion
1. Go to your [RudderStack dashboard](https://app.rudderstack.com/) and click **Add Source**. From the list of **Event Streams** sources, select **Monday**.
```

File: docs/sources/event-streams/cloud-apps/monday.mdx [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Monday"
+description: Step-by-step guide on ingesting your event data from monday.com into RudderStack.
+aliases: ["Monday source"]
+---
+
+[monday.com](https://monday.com/) is a popular workflow management tool. It lets you create efficient cross-team workflows and automations for better collaboration and maximum productivity.
```
Comment (ishachopra0401) [inline docs/sources/event-streams/cloud-apps/monday.mdx:7]:
```suggestion
[Monday.com](https://monday.com/) is a popular workflow management tool. It lets you create efficient cross-team workflows and automations for better collaboration and maximum productivity.
```

**Review by ishachopra0401 — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
@yashasvibajpai do we have an update on this? When is the source expected to be released?

**Comment by yashasvibajpai** [comment]
> @yashasvibajpai do we have an update on this? When is the source expected to be released?

@ameypv-rudder we are still caught with server release. Until then this would be on hold

**Review by yashasvibajpai — APPROVED** [review]

**Pull request merged**